config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += %w( govuk_admin_template/favicon.png )
```

Comment:
I think we should add [something like this too](https://github.com/alphagov/service-manual-publisher/blob/master/config/initializers/assets.rb#L13-L15), because [of this](https://github.com/alphagov/govuk_admin_template/blob/master/app/views/layouts/govuk_admin_template.html.erb#L24).